routes/routes.js

```diff
@@ -25,10 +25,13 @@ module.exports = function(app, passport) {
     })
   })
 
-  router.post('/create', function(req,res, next){
+  router.post('/:gameid/:userid', isLoggedIn, function(red,res,next){
```

Comment:
Oh shit hang on the changes in the routes.js are useless they're not actually doing anything.
I'm going to get rid of them entirely. I haven't set up any endpoints/routes for the functions to run, only made them,.